monitoring/README.md

```diff
@@ -46,6 +46,85 @@ metrics from the workers and provide the `/metrics` endpoint for Prometheus.
    Pushgateway] to be set up and configured for scraping.
 3. Monitor some resources.
 
+### How to monitor the Openshift objects?
+
+1. [`kube-state-metrics`](https://github.com/kubernetes/kube-state-metrics)
+
+   - converts Kubernetes objects to metrics consumable by Prometheus
+   - not focused on the health of the individual Kubernetes components, but rather on the health of the various objects inside, such as deployments, nodes and pods
+   - metrics are exported on the HTTP endpoint `/metrics` on the listening port, designed to be consumed either by
+     Prometheus itself or by a scraper that is compatible with scraping a Prometheus client endpoint
+   - deleted objects are no longer visible on the `/metrics` endpoint
+   - examples of exposed metrics (more [here](https://github.com/kubernetes/kube-state-metrics/tree/master/docs)):
+     - kube_pod_container_resource_requests
+     - kube_pod_container_status_restarts_total
+     - kube_pod_status_phase
+   - in limited-privileges environment where you don't have cluster-reader role:
```

Comment:
I hope they have clear docs how to set it up because roles in k8s are always a nightmare to figure out properly